### PR TITLE
Enabled ARM mode for NETWORK and BVT Test cases.

### PR DESCRIPTION
**Pull request by iamshital**

### CODE REVIEW

File: TestLibs/ARMLibrary.psm1 [hunk 1]
```diff
@@ -0,0 +1,935 @@
...
+            $groupName = $groupName + "-" + $resourceGroupCount
+        }
+
+        while (($isServiceDeployed -eq "False") -and ($retryDeployment -lt 5))
+        {
+            LogMsg "Creating Resource Group : $groupName."
+            LogMsg "Verifying that Resource group name is not in use."
+            #$isServiceDeleted = DeleteResourceGroup -RGName $groupName 
+			$isServiceDeleted = $true
+            if ($isServiceDeleted)
```
Comment (lizzha) [inline TestLibs/ARMLibrary.psm1]: clean up needed

File: TestLibs/RDFELibs.psm1 [hunk 2]
```diff
@@ -2304,69 +2371,53 @@ Function GetStopWatchElapasedTime([System.Diagnostics.Stopwatch]$sw, [string] $f
...
-	foreach ($hostedservice in $DeployedServices.Split("^"))
+	foreach ($testVM in $allVMData)
 	{
-		$DeployedVMs = Get-AzureVM -ServiceName $hostedService
-		foreach ($testVM in $DeployedVMs)
+		$HSVIP = $testVM.PublicIP
+		$HSport = $testVM.SSHPort
+		$testIP = $testVM.PublicIP
+		$testPort = $testVM.SSHPort
+		#$LisLogFile = "LIS-Logs-" + $testVM.InstanceName + ".tgz"
```
Comment (lizzha) [inline TestLibs/RDFELibs.psm1]: Remove it

File: TestLibs/ARMLibrary.psm1 (hunk 1 above)
Comment (iamshital) [inline TestLibs/ARMLibrary.psm1]: done.

File: TestLibs/RDFELibs.psm1 (hunk 2 above)
Comment (iamshital) [inline TestLibs/RDFELibs.psm1]: done.

**Pull request merged**